src/services/authService.js

```diff
@@ -30,11 +32,65 @@ export default class AuthService {
     return { token, ...Helper.omitFields(user, ['password']) };
   }
 
+  /**
+ *
+ * @param {string} email email address
+ * @returns {string} token
+ */
+  static async forgotPassword(email) {
+    const result = await User.findOne({ where: { email } });
+
+    if (!result) throw new Error('Email not found');
+
+    const { dataValues: { first_name: name, } } = result;
+    const token = await jwt.sign({
+      email
+    }, jwtSecret, { expiresIn: '1h' });
+    const emailDetails = {
+      receiver: email,
+      sender: process.env.SENDER,
+      templateName: 'reset_password',
+      name,
+      url: `reset_password/${token}`
```

Comment:
This url should be `${process.env.SERVICE_URL}/reset_password/${token}`
The `process.env.SERVICE_URL` can be anything for now but will change when we start working on our frontend